### PR TITLE
[RC1 Sodium] Update to the latest salt-bootstrap release, v2020.05.28

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

**Review by DmitryKuzmenko — APPROVED** [review]
Some tests failed but I don't see how it could be related. @s0undt3ch thoughts?

**Comment by s0undt3ch** [comment]
They shouldn't be

**Comment by s0undt3ch** [comment]
re-run all

**Comment by sagetherage** [comment]
re-run

File: salt/cloud/deploy/bootstrap-salt.sh [hunk 1]
```diff
@@ -7222,7 +7277,7 @@ install_macosx_stable_post() {
     set +o nounset
     # shellcheck disable=SC1091
     . /etc/profile
-    # Revert nounset to its previous state
+    # Revert nounset to it's previous state
```
Comment (mchugh19) [inline salt/cloud/deploy/bootstrap-salt.sh:7280]: nitpick, but this should be "its"

File: salt/cloud/deploy/bootstrap-salt.sh [hunk 2]
```diff
@@ -2620,7 +2629,7 @@ __install_tornado_pip() {
 #    PARAMETERS:  requirements_file
 #----------------------------------------------------------------------------------------------------------------------
 __install_pip_deps() {
-    # Install virtualenv to system pip before activating virtualenv if that's going to be used
+    # Install virtualenv to system pip before activating virtualenv if thats going to be used
```
Comment (mchugh19) [inline salt/cloud/deploy/bootstrap-salt.sh:2632]: nitpick: should be "that's"
Might be slightly clearer to just make it "it is"

File: salt/cloud/deploy/bootstrap-salt.sh [hunk 3]
```diff
@@ -1869,7 +1872,7 @@ if [ "$ITYPE" = "git" ]; then
     if [ "$_POST_NEON_INSTALL" -eq $BS_TRUE ]; then
         echo
         echowarn "Post Neon git based installations will always install salt"
-        echowarn "and its dependencies using pip which will be upgraded to"
+        echowarn "and it's dependencies using pip which will be upgraded to"
```
Comment (mchugh19) [inline salt/cloud/deploy/bootstrap-salt.sh:1875]: nitpick: should be "its"

File: salt/cloud/deploy/bootstrap-salt.sh [hunk 4]
```diff
@@ -349,7 +349,7 @@ __usage() {
         with -c and -k
     -C  Only run the configuration function. Implies -F (forced overwrite).
         To overwrite Master or Syndic configs, -M or -S, respectively, must
-        also be specified. Salt installation will be omitted, but some of the
+        also be specified. Salt installation will be ommitted, but some of the
```
Comment (mchugh19) [inline salt/cloud/deploy/bootstrap-salt.sh:352]: spelling nit: "omitted"

**Comment by s0undt3ch** [comment]
@mchugh19 Thanks for the review! Fixes should be done in the salt-bootstrap repo, so I'll try to remember these.

**Pull request merged**